examples/retinanet/model.py

```diff
@@ -0,0 +1,412 @@
+from anchor import *
```

Comment:
Could you instead import the module & then reference elements like `anchor.AnchorConfig` ?

https://google.github.io/styleguide/pyguide.html#22-imports